db/migrate/20230815110421_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :text
+      t.references :posts_id, null: false, foreign_key: true
+      t.references :user_id, null: false, foreign_key: true
+
+      t.timestamps
```

Comment:
- [OPTIONAL] KIndly make sure to rename the column `author` instead of `user` in order to match the provided [ERD DIAGRAM](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) and project requirement.


```suggestion
  def change
    create_table :comments do |t|
      t.string :text
      t.references :posts_id, null: false, foreign_key: true
     t.references :author, foreign_key: { to_table: 'users' }

      t.timestamps
```